### PR TITLE
Adding author field to disruption

### PR DESCRIPTION
**Pull request by tchevily**
# Description

This PR adds author field on disruption

## Issue (optional)

Issue link: BOT-1475

## Pull Request type (optional)

This is a new feature

### CODE REVIEW

File: chaos/history.py [hunk 1]
```diff
@@ -59,13 +58,16 @@ def create_disruption_from_json(json):
...
     disruption.impacts = create_impacts_from_json(json['impacts'], json['id'])
+    disruption.author = get_athor_from_json(json)
 
     return disruption
 
 
 def get_datetime_from_json_attr(json, attr):
     return parse_datetime(json[attr]).replace(tzinfo=None) if attr in json and json[attr] else None
 
+def get_athor_from_json(json):
```
Comment (pthegner) [inline chaos/history.py]:
```suggestion
def get_author_from_json(json):
```

File: chaos/history.py [hunk 2]
```diff
@@ -59,13 +58,16 @@ def create_disruption_from_json(json):
     disruption.tags = create_tags_from_json(json['tags'])
     disruption.properties = create_properties_from_json(json['properties'], json['id'])
     disruption.impacts = create_impacts_from_json(json['impacts'], json['id'])
+    disruption.author = get_athor_from_json(json)
```
Comment (pthegner) [inline chaos/history.py]:
```suggestion
    disruption.author = get_author_from_json(json)
```

File: chaos/history.py (hunk 1 above)
Comment (tchevily) [reply chaos/history.py]: Done

**Comment by kun2985** [comment]
`author` is missing in `POST /disruptions/_search` response

**Comment by pthegner** [comment]
> 
> 
> `author` is missing in `POST /disruptions/_search` response

I think it's on `POST /impacts/_search` response (in the disruption section)

**Comment by tchevily** [comment]
> > `author` is missing in `POST /disruptions/_search` response
> 
> I think it's on `POST /impacts/_search` response (in the disruption section)

Maybe later

**Comment by dvdn** [comment]
Ping @vpassama , just for information.

**Pull request merged**